pandas/core/frame.py

```diff
@@ -4168,14 +4168,14 @@ def dropna(self, axis=0, how='any', thresh=None, subset=None,
 
         Parameters
         ----------
-        axis : {0 or 'index', 1 or 'columns'}, or tuple/list thereof
+        axis : {0 or 'index', 1 or 'columns'}
```

Comment:
I believe the standard for this is `axis : {0 or ‘index’, 1 or ‘columns’, None}, default None`

See end of pandas docstring guide section on Parameters:

https://python-sprints.github.io/pandas/guide/pandas_docstring.html#section-3-parameters

Reply:
I've added the default option here. `dropna`, however, does not support `None` as an input to axis.

Reply:
Hmm if that's the case then just go with `axis : {0 or 'index', 1 or 'columns'}, default 0`. That's what I see everywhere else in the code anyway

@datapythonista - something we should update in the docstring standard?